css/style-sass-components.css

```diff
@@ -6364,28 +6362,46 @@ Style guide: page-header.5
   .page-header--xlarge-intro, .node-info-homepage.view-mode-homepage_political .page-header {
     height: 320px; }
     .page-header--xlarge-intro .page-header__hero-title, .node-info-homepage.view-mode-homepage_political .page-header .page-header__hero-title {
-      margin: auto;
-      padding-top: 5em;
-      background-color: transparent; }
-    .page-header--xlarge-intro .field__label--above, .node-info-homepage.view-mode-homepage_political .page-header .field__label--above {
-      width: 146px;
-      height: 34px;
-      margin: 0;
-      line-height: 34px;
-      background-color: #003776;
-      text-align: center; }
-    .page-header--xlarge-intro p, .node-info-homepage.view-mode-homepage_political .page-header p {
-      width: 550px;
-      padding: 22px 5px 22px 33px;
-      background-color: #fff;
-      color: #004494; } }
+      margin: 245px 0 0; } }
+
+@media (min-width: 768px) {
+  .page-header--xlarge-intro .page-header__hero-title, .node-info-homepage.view-mode-homepage_political .page-header .page-header__hero-title {
+    margin: auto;
+    padding-top: 5em;
+    background-color: transparent; }
+  .page-header--xlarge-intro .field__label--above, .node-info-homepage.view-mode-homepage_political .page-header .field__label--above {
+    width: 146px;
+    height: 34px;
+    margin: 0;
+    line-height: 34px;
+    background-color: #003776;
+    text-align: center; }
+  .page-header--xlarge-intro p, .node-info-homepage.view-mode-homepage_political .page-header p {
+    width: 550px;
+    padding: 22px 5px 22px 33px;
+    background-color: #fff;
+    color: #004494; } }
+
+.node-info-homepage.view-mode-homepage_political .page-header.page-header--image {
```

Comment:
Indeed, that should be the way, but this would prevent us from showing those demos that we currently have in the page header component in the styleguide.
Therefore, no action for now, we could tag with a @todo that section so that at least we know that it's supposed to be moved at some point.